docs/classes/ERC725.md

```diff
@@ -16,22 +14,20 @@ It is more convenient to use [`fetchData`](ERC725.md#fetchdata).
 It does the `decoding` and `fetching` of external references for you automatically.
 :::
 
-### Parameters
+#### Parameters
 
-| Name   | Type     | Description                            |
-| :----- | :------- | :------------------------------------- |
-| `data` | `Object` | An object with one or many properties. |
+1. `data` - `Object`: An object with one or many properties. The object's keys should match the key names defined in the `schema`, and the value should be the encoded value you want to decode. It will decode the value according to the corresponding schema.
 
-### Returns
+#### Returns
 
 `Object`
 
 Returns decoded data as defined and expected in the schema:
 
-### Example
+#### Example
 
 ```javascript title="Decode one key"
-const decodedDataOneKey = myERC725.decodeData({
+const decodedDataOneKey = erc725.decodeData({
```

Comment:
should this be `myERC725.decodeData()`? Or if it is a static method then `ERC725.decodeData()`